### PR TITLE
chore:made changes to card component

### PR DESCRIPTION
**Pull request by AceDheGeneral**
adjusted the spacing between plan name and download button in explore plan card

### CODE REVIEW

File: package-lock.json
Comment (Bendomey) [inline package-lock.json:1]: @arkhurst is he using yarn now?

Comment (arkhurst) [reply package-lock.json:1]: he's supposed to be using yarn. I think he was having an issue with it on his sub-system

Comment (Bendomey) [reply package-lock.json:1]: So he is using yarn now right? Then I guess we can remove this file!

**Review by arkhurst — COMMENTED** [review]
@AceDheGeneral going forward let's add screenshots of what you worked on to the PRs

File: package-lock.json
Comment (Bendomey) [reply package-lock.json:1]: @AceDheGeneral Please delete this file `package-lock.json` =)

Comment (AceDheGeneral) [reply package-lock.json:1]: ohk sure

**Review by Bendomey — APPROVED** [review]

**Comment by Bendomey** [comment]
Good job!

**Pull request merged**